src/client/content_cao.cpp

```diff
@@ -881,7 +881,7 @@ void GenericCAO::updateLight(u32 day_night_ratio)
 	if (!pos_ok)
 		light_at_pos = LIGHT_SUN;
 
-	video::SColor light = encode_light(light_at_pos, decode_light(m_prop.glow));
+	video::SColor light = encode_light(light_at_pos, m_prop.glow);
```

Comment:
`glow` is a value between [0, LIGHT_SUN] but `encode_light` treats the increment parameter as a value between [0, 255] for the effective brightness, hence the `decode_light` call here was more correct than the current code.

Reply:
I have added comments explaining how glow is used now:
* Glow now impacts light level of the entity in the getInteriorLight call
* A small boost is added to the final color to make entities brighter at the same light level